src/components/auth/Register.js

```diff
@@ -65,11 +65,10 @@ class Register extends React.Component {
               const userid = res2.data.id;
               localStorage.setItem("token", res2.data.token);
               this.props.updateToken(localStorage.getItem("token"));
-              this.props.history.push(
-                "/dashboard",
-                { userid: userid },
-                { pathname: "/dashboard" }
-              );
+              this.props.updateUserid(res2.data.id);
+              this.props.history.push(`/${res2.data.id}/recommended`, {
+                userid: userid
```

Comment:
Removed since using the match object instead of the history object.

Reply:
This is good, don't know if it was related but I was getting undefined in the previous iteration path name, which is not the case on this deployment